LICENSES/LicenseRef-Slint-Royalty-free-1.0.md

```diff
@@ -0,0 +1,37 @@
+# Slint Royalty-free Desktop and Web Applications License, version 1.0
```

Comment:
This looks like it overlaps with https://github.com/slint-ui/slint/pull/2884 . Did you mean to omit the addition of this file in this PR?

Reply:
changes done